components/eamxx/src/physics/ml_correction/eamxx_ml_correction_process_interface.cpp

```diff
@@ -12,10 +12,14 @@ MLCorrection::MLCorrection(const ekat::Comm &comm,
                            const ekat::ParameterList &params)
     : AtmosphereProcess(comm, params) {
   m_ML_model_path_tq = m_params.get<std::string>("ML_model_path_tq");
+  m_ML_model_path_t_only = m_params.get<std::string>("ML_model_path_temperature");
   m_ML_model_path_uv = m_params.get<std::string>("ML_model_path_uv");
   m_ML_model_path_sfc_fluxes = m_params.get<std::string>("ML_model_path_sfc_fluxes");
   m_fields_ml_output_variables = m_params.get<std::vector<std::string>>("ML_output_fields");
   m_ML_correction_unit_test = m_params.get<bool>("ML_correction_unit_test");
+  EKAT_ASSERT_MSG(!(m_ML_model_path_tq != "None" && m_ML_model_path_t_only != "None"),
```

Comment:
For some reason, I can't trigger this error in the full run but the same logic worked in the unit test. Maybe there's a better to handle this?